src/migrations/20230323000000-fei-template-goal.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable max-len */
```

Comment:
note that this also borks the goal seeder if you use the test-db shell script, as the migrations run first and creates a goal template with an ID of 1, the same ID which is hardcoded into the goal seeder

Reply:
I guess more importantly than that, it also borks the CI :)

Reply:
hmmmm ok I will need to figure that out